Topics/jupyter-notebook.md

```diff
@@ -0,0 +1,154 @@
+# Jupyter Notebook
```

Comment:
It's a good practise to expand on what the topic means, for e.g. 
JuPyTeR is a combined word made from the words Julia, Python and R